include/utils/grpchelper.hpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef GRPCHELPER_HPP_
+#define GRPCHELPER_HPP_
+
+#include <grpcpp/security/credentials.h>
+#include <grpcpp/security/server_credentials.h>
+
+#include <aos/common/crypto.hpp>
+#include <aos/common/cryptoutils.hpp>
+#include <aos/iam/certhandler.hpp>
+
+std::shared_ptr<grpc::ServerCredentials> GetMTLSServerCredentials(const aos::iam::certhandler::CertInfo& certInfo,
```

Comment:
It'll be great to add doxygen comments for all these public functions.

Reply:
Done